src/components/input/code/views/inputView.jsx

```diff
@@ -1,4 +1,5 @@
 var React = require('react');
+var _ = require('lodash');
```

Comment:
as above, require just the lodash method you require (unless you require all of lodash)